lib/stack_master/template_compilers/yaml.rb

```diff
@@ -0,0 +1,12 @@
+module StackMaster::TemplateCompilers
+  class Yaml
+
+    def self.compile(template_file_path)
+      template_body = File.read(template_file_path)
+      JSON.dump(YAML.load(template_body))
+    end
+
+    StackMaster::TemplateCompiler.register(:yaml, self)
+    StackMaster::TemplateCompiler.register(:yml, self)
```

Comment:
Yeah I think only the `:yaml` definition would be required... The file extension mapping is handled in config.